### PR TITLE
[zh-cn]: create the translation of `declarativeNetRequest.updateEnabledRulesets()` method

### PR DESCRIPTION
**Pull request by tianyeeT**
…edRulesets()` method

### CODE REVIEW

File: files/zh-cn/mozilla/add-ons/webextensions/api/declarativenetrequest/updateenabledrulesets/index.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+      - : 一个 `string` 数组。要禁用的静态规则集的 ID。
+    - enableRulesetIds {{optional_inline}}
+      - : 一个 `string` 数组。要激活的静态规则集的 ID。
+
+### 返回值
+
+一个 [`Promise`](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Promise)。如果请求成功，其会被兑现且不带参数。如果请求失败，则 promise 将被拒绝并带有错误消息。
+
+## 示例
+
```
Comment (yin1999) [inline files/zh-cn/mozilla/add-ons/webextensions/api/declarativenetrequest/updateenabledrulesets/index.md:35]: 应该去掉，下面的宏会自己渲染一个小标题（更新一下英文文档）

```suggestion
```

**Comment by tianyeeT** [comment]
Related PR: https://github.com/mdn/content/pull/34223

**Comment by tianyeeT** [comment]
> [mdn/content#34223](https://github.com/mdn/content/pull/34223)

Updates: https://github.com/mdn/content/pull/34223#pullrequestreview-2129185198

**Review by yin1999 — APPROVED** [review]
问题在于，不应该留一个空的标题在那里。上游建议更新宏的代码，那就先这样吧

**Pull request merged**